tests/libs/common/common_tests.cpp

```diff
@@ -189,17 +189,30 @@ void
 ring_buffer_api_test_helper(
```

Comment:
Also add a test case verifying that the API _fails_ if the caller passes an fd that is not an appropriate ring buffer map.